src/ops/ops.go

```diff
@@ -504,13 +506,98 @@ func (o *ops) RemovePV(pvName string) error {
 }
 
 func (o *ops) Wipefs(device string) error {
-	output, err := o.ExecPrivilegeCommand(o.logWriter, "wipefs", "-a", device)
+	_, err := o.ExecPrivilegeCommand(o.logWriter, "wipefs", "--all", "--force", device)
+
 	if err != nil {
-		o.log.Errorf("Failed to wipefs device %s, output %s, error %s", device, output, err)
+		_, err = o.ExecPrivilegeCommand(o.logWriter, "wipefs", "--all", device)
 	}
+
 	return err
 }
 
+func (o *ops) IsRaidDevice(device string) bool {
+	output, err := o.ExecPrivilegeCommand(o.logWriter, "mdadm", "-v", "--query", "--detail", "--scan")
+
+	if err != nil {
+		o.log.WithError(err).Errorf("Error occurred while trying to get list of raid devices - continue without cleaning")
+		return false
+	}
+
+	return strings.Contains(output, device)
```

Comment:
This might trip up on `/dev/sda1` and `/dev/sda12`, is it possible to do parsing that's a little more structured?

Reply:
That's ok, We want to do a quick check if the disk itself or one of its partitions are part of the raid array.

Reply:
https://github.com/torvalds/linux/blob/fe91c4725aeed35023ba4f7a1e1adfebb6878c23/drivers/scsi/sd.c#L3415-L3417